src/components/WalletModal/PendingView.tsx

```diff
@@ -78,7 +78,13 @@ export default function PendingView({
           return (
             <Flex key={key} mb="28px" justifyContent="center">
               <Box mr="10px">
-                <img src={require('../../assets/images/' + option.iconName)} alt="logo" width="24px" height="24px" />
+                <img
+                  // eslint-disable-next-line
+                  src={require('../../assets/images/' + option.iconName).default}
```

Comment:
this might be cleaner as function, something like: `iconImageSrc(iconName)`

And could be something like this:
`iconImageSrc = (icon) => require('../../assets/images/' + option.iconName).default`

Probably we can get rid of the eslint disable